### ISSUE TITLE
External sites are not working after the upgrade to 27.x.x

### ISSUE THREAD

**Issue by ostasevych** [nextcloud/android#12095]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

1. Try to open any external site created by administrator, eg `/apps/external/1`
2. Observe that it is opening through webUI
3. Observe that in the mobile version instead of the page with the external site I see the main page of the UI: file navigation

### Expected behaviour

The page with the external site should be shown, as it has been before upgrading the server side to 27.x.x

### Actual behaviour

Instead of the external page I observe the file navigation page

### Android version

14

### Device brand and model

Pixel 5A

### Stock or custom OS?

Stock

### Nextcloud android app version

3.26.0

### Nextcloud server version

27.1.2

### Using a reverse proxy?

I don't know

**Comment by joshtrichards** [comment]
Related: nextcloud/external/issues/577

**Comment by ostasevych** [comment]
Yes, I've added the issues on both the app and android github pages in order to capture the attention of both developers.

**Comment by Naeaeaeaeae** [comment]
I have external sites, used as weblinks and the android App does not refresh the list oft external sites, but the iOS App does refresh.

-> add a new external sites -> Android APp does not pick it up.
